minichain/chain.py

```
@@ -47,18 +47,18 @@ def _create_genesis_block(self, genesis_path):
                 with open(genesis_path, "r") as f:
                     config = json.load(f)
             except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)